frontend/src/Pages/AdminPage.jsx

```
@@ -0,0 +1,334 @@
+                            ) : (
+                              <>
+                                <button
+                                  className="text-blue-500 hover:text-blue-700"
+                                  onClick={() => handleEditUser(user)}
+                                >
+                                  <Edit3 size={18} />
+                                </button>
+                                <button
+                                  className="text-red-500 hover:text-red-700"
+                                  onClick={() => handleDeleteUser(user._id)}
+                                  disabled={user._id === user._id}
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)